test/e2e/features/app/login.feature

```diff
@@ -1,13 +1,24 @@
 Feature: Do stuff
 
+    @all
```

Comment:
Don't need an `@all` tag. Running the tests without any tags will run the entire suite